### DISCUSSION TITLE
[QUESTION] calculate_skew: delta - skew: 0:00:01.010868203 too big, reset skew Error

### DISCUSSION THREAD

**Discussion by deorwine** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#1138] [Q&A]
i have used same RTSP URL working on other ipcamlive.com but when i am trying to kvssink it getting below error.
media info => 
h264
buffer dimention => 1920x1090
framerate => 29.97

anybody help what an issue.

-------------------------------------------------------------------

2024-01-24 13:34:14 [140222360712768] DEBUG - defaultStreamStateTransitionHook(): Stream state machine retry count: 2
2024-01-24 13:34:14 [140222360712768] DEBUG - defaultStreamStateTransitionHook(): 
 KinesisVideoStream base result is [599]. Executing KVS retry handler of retry strategy type [1]
2024-01-24 13:34:14 [140222360712768] INFO - putStreamResultEvent(): Put stream result event. New upload handle 1
DEBUG - postReadCallback(): [camera_101] Pausing CURL read for upload handle: 1
2024-01-24 13:34:17 [140222520075840] INFO - writeHeaderCallback(): RequestId: f8fc7664-e4ec-5f9c-a6d7-627c454b287d
DEBUG - Dropping frame with flag: 1536 for camera_10199:99.
0:00:49.591830151 307459 0x56012d25e180 WARN         rtpjitterbuffer rtpjitterbuffer.c:575:calculate_skew: delta - skew: 0:00:01.010868203 too big, reset skew
DEBUG - Dropping frame with flag: 1536 for camera_101
DEBUG - Dropping frame with flag: 1536 for camera_10199:99.
0:00:54.603249335 307459 0x56012d25e180 WARN         rtpjitterbuffer rtpjitterbuffer.c:575:calculate_skew: delta - skew: 0:00:01.029800347 too big, reset skew
DEBUG - Dropping frame with flag: 1536 for camera_101
0:01:00.780049553 307459 0x56012d25e180 WARN         rtpjitterbuffer rtpjitterbuffer.c:575:calculate_skew: delta - skew: 0:00:01.293152373 too big, reset skew
DEBUG - Dropping frame with flag: 1536 for camera_101
DEBUG - Dropping frame with flag: 1536 for camera_10199:99.
DEBUG - Dropping frame with flag: 1536 for camera_10199:99.
DEBUG - Dropping frame with flag: 1536 for camera_10199:99.
DEBUG - Dropping frame with flag: 1536 for camera_10199:99.

**Comment by niyatim23** [comment]
Hi @deorwine, can you modify your gstreamer pipeline to use filesink instead of kvssink and see if the same problem persists? This will help us rule out issues with kvssink
